recipes/interpret-core/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "interpret-core" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/interpret-core-{{ version }}.tar.gz
+  sha256: b0b0b9aed8214fc1c0e3c400eafd663db4d37dfd285d202d8339886c44e1b901
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - nodejs
+  host:
+    - pip
+    - python
+    - wheel
+    - compilers
```

Comment:
```suggestion
```

Reply:
Then there is something wrong in their scripts. They should reference `${CC}` not `gcc` explicitly.